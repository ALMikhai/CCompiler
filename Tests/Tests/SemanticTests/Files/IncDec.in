int f() {
    int a = 1;
    int b = a++;
}
