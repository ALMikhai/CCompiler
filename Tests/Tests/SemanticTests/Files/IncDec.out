\-TRANSLATIONUNIT
  \-FUNC
    |-DECLSPEC
    | |-INT
    |-INITDECLARATOR
    | \-DECLARATOR
    |   \-FUNC
    |     |-f
    |     \-EMPTY
    \-COMPOUNDSTAT
      |-DECLLIST
      | |-DECL
      | | |-DECLSPEC
      | | | |-INT
      | | \-INITDECLARATORLIST
      | |   \-INITDECLARATOR
      | |     |-DECLARATOR
      | |     | \-a
      | |     \-1
      | |-DECL
      | | |-DECLSPEC
      | | | |-INT
      | | \-INITDECLARATORLIST
      | |   \-INITDECLARATOR
      | |     |-DECLARATOR
      | |     | \-b
      | |     \-POSTFIX INC
      | |       \-a
      | \-DECL
      |   |-DECLSPEC
      |   | |-INT
      |   \-INITDECLARATORLIST
      |     \-INITDECLARATOR
      |       |-DECLARATOR
      |       | \-c
      |       \-PREFIX INC
      |         \-a

Structs{
}
Symbols{
	FUNC returning INT
	Arguments{
	}
	Structs{
	}
	Symbols{
		INT :: a
		INT :: b
		INT :: c
	} :: f
}
